Cargo.toml

```
@@ -28,7 +28,7 @@ thread_local = "~1.0"
 
 [dev-dependencies]
 clap = "~2.33.0"
-docopt = "~1.0"
+docopt = "~1.1"
```

```suggestion
docopt = "1.1"
```

For the dev dependencies, I think we can remove the tilde since it's really up to the end user to ensure their environment is good for the the examples for what they're trying to target.